### PR TITLE
Update LIKE data.json (update token symbol to match contract)

### PR DESCRIPTION
**Pull request by ENSpunks**
The admins changed the folder name from “❤️” to “LIKE” because the emoji as a folder name was problematic for the repo, but when the folder name was change the admins also changed the token symbol in error, this PR  edits the token symbol back to “❤️” consistent with both the Ethereum Mainnet and Base contracts.

**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Review by wbnns — APPROVED** [review]
ACK

**Comment by ENSpunks** [comment]
Hi, looks like my PR was blocked.

It’s just updating the token symbol to match the contracts.

anything I can/should do?

**Review by wbnns — CHANGES_REQUESTED** [review]
@ENSpunks 

Hello, please update, to fix the CI/validation errors.

**Comment by ENSpunks** [comment]
Hi, see below there no logs for this, and no error handling (since it’s uncommon).  Again the original PR and merge the token symbol was correct and I just received an email one day saying it was changed, can they just update it or revert it?

really not sure 

image

**Review by wbnns — CHANGES_REQUESTED** [review]
@ENSpunks 

Heya, thanks for being responsive. Here's what's causing the PR to fail:
```
error: final token list is invalid
warning: LIKE on chain ethereum token 0x0469dFb82A816C5F0a566625Cec09216A721E90e has overridden symbol
warning: LIKE on chain ethereum token 0x0469dFb82A816C5F0a566625Cec09216A721E90e not found on CoinGecko token list
warning: LIKE on chain base token 0xE0742c2d2Bdbb3c138Ce81Bc5589DC07865f0a49 has overridden symbol
```

**Comment by ENSpunks** [comment]
Does this sound right? 

The tokenlist “overriding” was either  done manually by the maintainers  when they changed the token symbol or it was done automatically when they did the change.

sorry I’m not technical and don’t know how to remove the over right that got added when the token symbol was change by the maintainers from “❤️” to “Like”

**Review by wbnns — CHANGES_REQUESTED** [review]
@ENSpunks 

Can remove the overrides from the JSON and also updating the root symbol to the heart emoji? I think that might be causing issues with the CI since on Etherscan and Basescan the symbol is "❤️"

**Review by wbnns — DISMISSED** [review]
@ENSpunks 

Can remove the overrides from the JSON and also updating the root symbol to the heart emoji? I think that might be causing issues with the CI since on Etherscan and Basescan the symbol is "❤️"

**Comment by wbnns** [comment]
Apologies for the double-post, looks like there might have been an intermittent issue with GitHub when I added the last comment.

**Comment by ENSpunks** [comment]
@wbnns 

thank you, overrides removed, I believe it will validate now

**Review by wbnns — DISMISSED** [review]
@ENSpunks 

Thanks for the additional update. Can you also please add the `"nonstandard": true,` option to your JSON?

**Comment by ENSpunks** [comment]
@wbnns 

Added    "nonstandard": true,

**Review by wbnns — CHANGES_REQUESTED** [review]
@ENSpunks 

Thanks for the update. Looks like there is still a validation issue.
```
error: final token list is invalid
```

**Comment by ENSpunks** [comment]
I think I figured it out, I added a new commit to the PR to update the tokenlist in addition to the data.json

**Comment by wbnns** [comment]
@ENSpunks 

Thanks for the update, I think the issue is that the validation script doesn't support emojis.

Cc: @hamdiallam

**Comment by hamdiallam** [comment]
Ah yes sorry for not chiming in earlier. This validation script was failing when checking against uniswap's tokenlist schema

I double checked and it looks like the latest uniswap schema supports unicode for the symbol. I put up #822. When that's merged in can you rebase and we'll get your token merged in.

I'd pref our folder names to not have any emoji symbols. The override approach here feels appropriate

File: data/LIKE/data.json [hunk 1]
```diff
@@ -1,22 +1,17 @@
 {
   "name": "LikeButton.eth",
-  "symbol": "LIKE",
```
Comment (hamdiallam) [inline data/LIKE/data.json:3]: I believe this needs to remain `LIKE` so that it matches the folder name

**Review by hamdiallam — DISMISSED** [review]

File: data/LIKE/data.json (hunk 1 above)
Comment (ENSpunks) [reply data/LIKE/data.json]: Looks like #822 was merged.

I changed the symbol back to “LIKE” to match the repo folder and added the overrides back in as “❤️” for base and ethereum to match the contract symbols.  

Please let me know if this isn’t correct

Comment (wbnns) [reply data/LIKE/data.json]: @ENSpunks Thanks!

@hamdiallam Looks like the CI is still failing because of the heart emoji. 🤔

**Comment by hamdiallam** [comment]
hey @ENSpunks can you rebase this PR so you pickup the changes in #822?

**Comment by ENSpunks** [comment]
I think the easiest way for me to rebase would be close this PR, delete my copy of the repo, fork the repo again and open a new PR.  Is that ok?

**Comment by wbnns** [comment]
@ENSpunks Sure, if you want to! Or if you want to give rebasing a go, feel free to try it out!

**Pull request closed without merging**